File: server/app/user_tests/user_test.py
def reverseWords(s: str) -> str:
    # Your code goes here 
  return  ' '.join(input_string.split()[::-1])

class SolutionTest:
    @staticmethod
    def run_test_case(input, expected):
        result = reverseWords(input)
        return result == expected

def run_all_tests():
    test_suite = SolutionTest()
    test_results = []
    test_cases = [
        {'input': "the sky is blue", 'expected': "blue is sky the"},
        {'input': "  hello world  ", 'expected': "world hello"},
        {'input': "a good   example", 'expected': "example good a"}
    ]
    for i, test_case in enumerate(test_cases, start=1):
        input, expected = test_case['input'], test_case['expected']
        result = test_suite.run_test_case(input, expected)
        test_results.append((f'Test case {i}', result))
    return test_results

if __name__ == '__main__':
    results = run_all_tests()
    for test_case, result in results:
        print(f'{test_case}: {True if result else False}')

File: server/app/user_tests/user_test.js
function removeDuplicates(array) {
    // Your code goes here
    return "ASDFASDASD"
}


function runTests() {
    const testResults = {}

    const testCases = [
        {input: [1,1,2], expected: 2},
        {input: [0,0,1,1,1,2,2,3,3,4], expected: 5},
        {input: [1,2,3], expected: 3}
    ];

    function arraysEqual(a, b) {
        return a.length === b.length && a.every((value, index) => value === b[index]);
    }

    testCases.forEach(({ input, expected }, index) => {
        const result = removeDuplicates(input);
        testResults[`testCase${index + 1}`] = {userOutput: result, expected: expected, assert: arraysEqual(result, expected)}
    });

    return JSON.stringify(testResults)
}

const testResults = runTests()
console.log(testResults)
